### PR TITLE
Finished working on the people search page

### PR DESCRIPTION
**Pull request by Iamronaldosunmu**

### CODE REVIEW

File: components/newsfeed/index.js [hunk 1]
```diff
@@ -3,14 +3,22 @@ import Logo from "../../components/Logo";
 import DarkModeToggle from "../../components/DarkModeToggle";
 import Link from "next/link";
 import HomeIcon from "../icons/HomeIcon";
+import { useState } from "react";
```
Comment (nightsailor) [inline components/newsfeed/index.js:7]: import FullNav from component/people/FullNav

**Comment by nightsailor** [comment]
Pls format all files in the directory components/people/, `npm run fileFormat components/people/` and other changed files.

File: components/people/PeopleSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import LookingForSelect from "./LookingFor";
+import PeopleSearchPerson from "./PeopleSearchPerson";
+import NotifEmpty from "../NotifEmpty";
+
+const PeopleSearch = ({people, searchValue}) => {
+    const reducedPeopleObj = people.slice(0, 5);
+    const [peopleObj, setPeopleObj] = useState(reducedPeopleObj);
+
+    const [searchResults, setSearchResults] = useState([]);
+    console.log(people.filter(person => person.name.includes(searchValue)));
```
Comment (nightsailor) [inline components/people/PeopleSearch.tsx:15]:
```suggestion

```

File: components/people/PeopleSearch.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    useEffect(() => {
+        const filtered = people.filter(person => person.name.toLowerCase().includes(searchValue.toLowerCase()));
+        setSearchResults(filtered);
+    }, [searchValue])
+  
+    const handleShowMore = () => {
+        setPeopleObj([...peopleObj, ...people.slice(5)]);
```
Comment (nightsailor) [inline components/people/PeopleSearch.tsx:23]:
```suggestion
        setPeopleObj(people);
```

File: components/people/PeoplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+const PeoplePage = ({people, setShowMessage}) => {
+    const verifiedPeopleObj = people.filter(person => person.verified === true);
+
+    const reducedPeopleObj = verifiedPeopleObj.slice(0, 6);
+    const [peopleObj, setPeopleObj] = useState(reducedPeopleObj);
+  
+    const unVerifiedPeopleObj = people.filter(person => person.verified !== true);
+
+    const handleShowMore = () => {
+        setPeopleObj([...verifiedPeopleObj, ...unVerifiedPeopleObj]);
```
Comment (nightsailor) [inline components/people/PeoplePage.tsx:15]:
```suggestion
        setPeopleObj(people);
```
The current code is okay, either ways

File: components/people/PeopleSearch.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
+import { useContext, useEffect, useState } from "react";
+import FullNav from "../newsfeed/index";
+import ArrowIcon from "../icons/Arrow";
+import MessageForm from "../MessageForm";
```
Comment (nightsailor) [inline components/people/PeopleSearch.tsx:4]: Pls remove all unused import's

Comment (Iamronaldosunmu) [reply components/people/PeopleSearch.tsx:4]: Okay👍

**Pull request merged**